contracts/CBKLandSale.sol

```diff
@@ -745,18 +745,18 @@ contract CBKLandSale is Initializable, AccessControlUpgradeable {
             require(reservedTier == TIER_TWO || reservedTier == TIER_THREE, "NA");
 
             uint256 counter = reservedChunksCounter[reseller];
-            for(uint256 i = counter; i < counter + rcLength; i++) {
-                uint256 cId = reservedChunks[reseller].at(uint256(i % rcLength));
+            for(uint256 j = counter; j < counter + rcLength; j++) {
```

Comment:
Are you sure it's going to work? Obviously your change makes sense, but it must have been working earlier I guess and this change looks like it would change the flow. @bilbolPrime is this something you worked on?

Reply:
The compiler warning is saying that you have "two `i`s" where the latter one hides the former one for the scope of the `for` loop.  My change merely gives a different name to the inner one so that it's not as surprising that they are actually two different variables on the stack.

Reply:
I am good with this change